packages/now-go/index.ts

```diff
@@ -1,24 +1,24 @@
-const { join, sep, dirname } = require('path');
-const {
+import { join, sep, dirname } from 'path';
+import {
   readFile, writeFile, pathExists, move,
-} = require('fs-extra');
+} from 'fs-extra';
 
-const glob = require('@now/build-utils/fs/glob.js'); // eslint-disable-line import/no-extraneous-dependencies
-const download = require('@now/build-utils/fs/download.js'); // eslint-disable-line import/no-extraneous-dependencies
-const { createLambda } = require('@now/build-utils/lambda.js'); // eslint-disable-line import/no-extraneous-dependencies
-const getWritableDirectory = require('@now/build-utils/fs/get-writable-directory.js'); // eslint-disable-line import/no-extraneous-dependencies
-const { createGo, getExportedFunctionName } = require('./go-helpers');
+import {
+  glob, download, createLambda, getWriteableDirectory, BuildOptions
+} from '@now/build-utils';
 
-const config = {
+import { createGo, getExportedFunctionName } from './go-helpers';
+
+export const config = {
   maxLambdaSize: '10mb',
 };
 
-async function build({ files, entrypoint }) {
+export const build = async ({ files, entrypoint }: BuildOptions) => {
```

Comment:
```suggestion
export async function build({ files, entrypoint }: BuildOptions) {
```

Let's keep this as a function declaration